resource access update (#127)
DavidXanatos/sandboxie-docs: Content/NoRenameWinClass.md

# No Rename Win Class

_NoRenameWinClass_ is a sandbox setting in [Sandboxie Ini](SandboxieIni.md). It specifies the window class names that should not be translated by Sandboxie.

> Examples:

Reviewer: ```suggestion
Usage:
```

```
   .
   .
   .
   [DefaultBox]
   NoRenameWinClass=ExampleWinClass
   NoRenameWinClass=program.exe,*
```

> The first setting tells Sandboxie to not translate _ExampleWinClass_ window class name, and also makes accessible to sandboxed programs, and goes a step further to disable a few other windowing-related Sandboxie functions. This may also cause the Sandboxie indicator [#] to not appear in window titles.

Reviewer: ```suggestion
The first setting tells Sandboxie to not translate _ExampleWinClass_ window class name by making it accessible to sandboxed programs, and goes a step further to disable a few other windowing-related Sandboxie functions. This may also cause the Sandboxie indicator [#] to not appear in window titles.
```

```

>    NoRenameWinClass=Program.exe,*

Reviewer: ```suggestion
   NoRenameWinClass=program.exe,*
```

```

> This setting tells Sandboxie to not translate window class names created by _Program.exe_ by making them accessible to sandboxed programs, and goes a step further to disable a few other windowing-related Sandboxie functions. This may also cause the Sandboxie indicator [#] to not appear in window titles.

Reviewer: ```suggestion
This setting tells Sandboxie to not translate window class names created by _program.exe_ by making them accessible to sandboxed programs, and goes a step further to disable a few other windowing-related Sandboxie functions. This may also cause the Sandboxie indicator [#] to not appear in window titles.
```

> 
> ```
>    NoRenameWinClass=program.exe,*
> ```
> 
> This setting tells Sandboxie to not translate window class names created by _program.exe_ by making them accessible to sandboxed programs, and goes a step further to disable a few other windowing-related Sandboxie functions. This may also cause the Sandboxie indicator [#] to not appear in window titles.

Reviewer: ```suggestion

The second setting tells Sandboxie to not translate window class names created by _program.exe_ by making them accessible to sandboxed programs, and goes a step further to disable a few other windowing-related Sandboxie functions. This may also cause the Sandboxie indicator [#] to not appear in window titles.
```

> This setting tells Sandboxie to not translate window class names created by _Program.exe_, and also makes accessible to sandboxed programs, and goes a step further to disable a few other windowing-related Sandboxie functions. This may also cause the Sandboxie indicator [#] to not appear in window titles.

Reviewer: ```suggestion
This setting tells Sandboxie to not translate window class names created by _Program.exe_ by making them accessible to sandboxed programs, and goes a step further to disable a few other windowing-related Sandboxie functions. This may also cause the Sandboxie indicator [#] to not appear in window titles.
```
